src/main/java/com/google/api/codegen/py/PythonImportHandler.java

```diff
@@ -60,7 +60,7 @@ public PythonImportHandler(Interface service) {
     // Add method request-type imports.
     for (Method method : service.getMethods()) {
       addImport(
-          method.getFile(),
+          method.getInputMessage().getFile(),
```

Comment:
The original is actually incorrect, but the problem didn't surface until I added mixin support. Basically, with mixins, the file declaring the input message type can be distinct from the file declaring the RPC; the type here needs to match the file of the input message type.